jenkins/manila-tempest/10-configure-nfs-server.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -xue
+
+if [[ ! ${USE_SCALITY_IMPL:-} ]]; then
+    USE_SCALITY_IMPL=1
+    echo "Using ${USE_SCALITY_IMPL} as default value for 'USE_SCALITY_IMPL'"
+fi
+
+if [ $USE_SCALITY_IMPL ]; then
```

Comment:
So if I export USE_SCALITY_IMPL=no or USE_SCALITY_IMP=0 it could have some kind of unexpected results.